bitnami/clickhouse/values.yaml

```diff
@@ -35,12 +35,15 @@ global:
       ## @param global.compatibility.openshift.adaptSecurityContext Adapt the securityContext sections of the deployment to make them compatible with Openshift restricted-v2 SCC: remove runAsUser, runAsGroup and fsGroup and let the platform use their allowed default IDs. Possible values: auto (apply if the detected running cluster is Openshift), force (perform the adaptation always), disabled (do not perform adaptation)
```

Comment:
There is a DEPRECATED value, `storageClass`

Reply:
done at https://github.com/bitnami/charts/pull/32958/commits/786dc3e86a09575f9fb564f1362f6c4b56c4e837